app/controllers/drafts_controller.rb

```diff
@@ -31,15 +29,13 @@ def index
           end
         end
       end
-
-      help_key += ".self"
     else
-      help_key += ".others"
+      raise Discourse::InvalidAccess
```

Comment:
:bear: with me during this review but I'll prefer that we raise the error as early as possible. IMO, the check and error can be raised after `user = fetch_user_from_params`